dtests/src/test/java/io/snappydata/hydra/connectionPool/TomcatConnectionPool.java

```diff
@@ -0,0 +1,69 @@
+
+/*
+ * Copyright (c) 2017 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package io.snappydata.hydra.connectionPool;
+
+
+import java.sql.Connection;
+
+import hydra.Log;
+import io.snappydata.hydra.cluster.SnappyTest;
+import org.apache.tomcat.jdbc.pool.DataSource;
+import org.apache.tomcat.jdbc.pool.PoolProperties;
+
+public class TomcatConnectionPool {
+
+  private static TomcatConnectionPool connPoolInstance = null;
+  public DataSource datasource;
+
+  private TomcatConnectionPool() {
+    Log.getLogWriter().info("[Sonal] Creating instance of TomcatConnectionPool");
```

Comment:
Remove unnecessary log statement.

Reply:
I will remove the name from the log statement, it was accidentally checked-in. But, this log statement is necessary for debugging purpose, hence I have kept it.